brat/Data/Hugr.hs

```diff
@@ -164,7 +169,48 @@ hugrInt = HTOpaque "arithmetic.int.types" "int" [TANat intWidth] TBEq
 hugrFloat :: HugrType
 hugrFloat = HTOpaque "arithmetic.float.types" "float64" [] TBCopy
 
-------------------------------------- OPS --------------------------------------
+
+------------------------------------ VALUES ------------------------------------
+-----------------------  (Depends on Hugr and HugrType)  -----------------------
+
+-- Depends on `Hugr` (Maybe it should be parametric in the argument to `Hugr`?)
+-- And on `HugrType` (for `HVExtension`)
+data HugrValue
+ = HVFunction (Hugr Int)
+ | HVTuple [HugrValue]
+ | HVExtension [ExtensionName] HugrType CustomConst
+ deriving (Eq, Show)
+
+instance ToJSON HugrValue where
+  toJSON (HVFunction h) = object ["v" .= ("Function" :: Text)
+                                 ,"hugr" .= h
+                                 ]
+  toJSON (HVTuple vs) = object ["v" .= ("Tuple" :: Text)
+                                  ,"vs" .= vs
+                                  ]
+  toJSON (HVExtension exts ty val) = object ["v" .= ("Extension" :: Text)
+                                            ,"typ" .= ty
+                                            ,"value" .= val
+                                            ,"extensions" .= exts
+                                            ]
+
+hvUnit = HVTuple []
+hvFloat x = HVExtension ["arithmetic.float_types"] hugrFloat
+            (CC "ConstF64" (KeyMap.singleton "value" x))
+-- Default to using 64-bit ints. At some point we need to update this so that
+-- we don't need to worry about int size when writing BRAT code. To do this, we
+-- need to add a bunch of widening operations during compilation.
```

Comment:
Why would we need to update that? Hugr ints are capped at 64 bits:

https://github.com/CQCL/hugr/blob/f2ae52ea869a639f15be88ebae8132e3a24c01d7/hugr-core/src/std_extensions/arithmetic/int_types.rs#L58-L59

Reply:
It seems like an unnecessary restriction, but since it's a hard limit in hugr, I'll drop the comment